packages/engine/tests/units/message/stop_simulation/integration-test.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "steps": 5,
+    "expected-output": {}
```

Comment:
This test is ignored but if we fix stop messages this file won't actually be checking anything like this, right?

Reply:
I'm still thinking on how to test this test. With the new testing strategy (don't call the CLI but the backing library) this may be easier to `.unwrap_err`.